ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ToolkitConfigurationService } from 'toolkit';
+import { Auth0AdapterService, InvitationServiceAgent } from 'ddp-sdk';
+import { ErrorType } from '../../models/errorType';
+import { take } from 'rxjs/operators';
+
+@Component({
+  selector: 'user-registration-prequal',
+  templateUrl: './user-registration-prequal.component.html',
+  styleUrls: ['./user-registration-prequal.component.scss']
```

Comment:
I split component into html and scss files, because it's extremely convenient to use html and scss tips and formatting.

Reply:
Sure! I personally prefer to keep it all in one file as I find it easier to keep it all together, but IDE does not really help me much, so happy to split things up.